kitsune/upload/storage.py

```diff
@@ -3,7 +3,10 @@
 import os
 import time
 
-from django.core.files.storage import FileSystemStorage as DjangoStorage
+try:
+    from storages.backends.s3boto3 import S3Boto3Storage as DjangoStorage
```

Comment:
It will use S3 in local instance?

Reply:
Only if you have django-storages and boto3 installed. We could also add a setting when this moves to the docker/master branch.